src/components/GlobalStyle.ts

```diff
@@ -17,4 +17,8 @@ export const GlobalStyle = createGlobalStyle`
 
         background-color: ${backgroundColor};
     }
+
+    * {
```

Comment:
No way) Don't use global overrides plz

Reply:
move this style to FormInput and FormTextarea